### PR TITLE
test: test karstnft

### PR DESCRIPTION
**Pull request by Jemiiah**

### CODE REVIEW

File: tests/test_karstnft.cairo [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    let mut calldata: Array<felt252> = array![ADMIN];
+    names.serialize(ref calldata);
+    symbol.serialize(ref calldata);
+    base_uri.serialize(ref calldata);
+    let (nft_contract_address, _) = nft_contract.deploy(@calldata).unwrap_syscall();
+    (nft_contract_address)
+}
+
+#[test]
+fn test_nft_count_on_init_is_zero() {
```
Comment (Darlington02) [inline tests/test_karstnft.cairo:29]: in here, I expected you'd be checking that the total supply on deployment is zero, not the balance of a user

File: tests/test_karstnft.cairo [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+    let last_minted_id = dispatcher.get_last_minted_id();
+
+    assert(last_minted_id.is_zero(), 'last minted id not zero');
+
+    stop_prank(CheatTarget::One(nft_contract_address));
+}
+
+#[test]
+fn test_user_token_id_on_init_is_zero() {
```
Comment (Darlington02) [inline tests/test_karstnft.cairo:59]: you can delete this test case, not useful imo

File: tests/test_karstnft.cairo [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+    let symbol: ByteArray = "KNFT";
+    let base_uri: ByteArray = "ipfs://QmSkDCsS32eLpcymxtn1cEn7Rc5hfefLBgfvZyjaYXr4gQ/";
+    let mut calldata: Array<felt252> = array![ADMIN];
+    names.serialize(ref calldata);
+    symbol.serialize(ref calldata);
+    base_uri.serialize(ref calldata);
+    let (nft_contract_address, _) = nft_contract.deploy(@calldata).unwrap_syscall();
+    (nft_contract_address)
+}
+
```
Comment (Darlington02) [inline tests/test_karstnft.cairo:27]: add two additional test cases to check for the name, symbol of the NFT

**Review by Darlington02 — CHANGES_REQUESTED** [review]

File: tests/test_karstnft.cairo (hunk 3 above)
Comment (Jemiiah) [reply tests/test_karstnft.cairo:27]: Thank you will make necessary changes

File: tests/test_karstnft.cairo (hunk 1 above)
Comment (Jemiiah) [reply tests/test_karstnft.cairo]: I checked the ERC721 component interface on openzeppelin but I couldn't find a function that returns the total supply. Do you have an idea of how to get it?

**Comment by Darlington02** [comment]
@allcontributors please add @Jemiiah as contributor for code

**Pull request merged**